apps/invoke-groups/generate.mjs

```diff
@@ -24,7 +24,7 @@ const notice = `
 
 const baseDir = new URL('.', import.meta.url).pathname;
 const templateDir = path.join(baseDir, 'template');
-const targetDir = path.join(baseDir, 'overlays/production');
+const targetDir = path.join(baseDir, 'overlays/production/generated');
```

Comment:
Are these now under the `/generated` directory, because they are automatically generated?

Reply:
Yep, thought it's best to isolate the ones generated from `template` from the custom ones.